### DISCUSSION TITLE
Firebird_ODBC Download Link not working

### DISCUSSION THREAD

**Discussion by asperand87** [FirebirdSQL/firebird#7740] [General]
hello,
i'm trying to download 
[Firebird_ODBC_2.0.4.155_Win32.exe](https://sourceforge.net/projects/firebird/files/firebird-ODBC-driver/2.0.4-Release/Firebird_ODBC_2.0.4.155_Win32.exe/download) 
or 
[Firebird_ODBC_2.0.4.155_x64.exe](https://sourceforge.net/projects/firebird/files/firebird-ODBC-driver/2.0.4-Release/Firebird_ODBC_2.0.4.155_x64.exe/download)
or any other release.
downloads do not start.

could you please help me?

kind regards

Angelo

**Comment by mrotteveel** [comment]
On 07-09-2023 08:29, asperand87 wrote:
> hello,
> i'm trying to download
> Firebird_ODBC_2.0.4.155_Win32.exe 
> <https://sourceforge.net/projects/firebird/files/firebird-ODBC-driver/2.0.4-Release/Firebird_ODBC_2.0.4.155_Win32.exe/download>
> or
> Firebird_ODBC_2.0.4.155_x64.exe 
> <https://sourceforge.net/projects/firebird/files/firebird-ODBC-driver/2.0.4-Release/Firebird_ODBC_2.0.4.155_x64.exe/download>
> or any other release.
> downloads do not start.
> 
> could you please help me?

This is a problem at SourceForge. According to 
***@***.*** they are currently having 
issues with some services, and I guess this includes their downloads.

The project no longer uses SourceForge as the primary hosting of 
binaries, but given there have been no release of the ODBC driver in a 
long time, there are none available at GitHub.

I would suggest to try it again later.

Mark

PS. The latest version of the ODBC driver is 2.0.5, so why try to 
download 2.0.4?
-- 
Mark Rotteveel

**Reply by asperand87** [reply]
thank you, same for 2.0.5 .
any other idea?
thankyou

**Reply by mrotteveel** [reply]
As I said, SourceForge seems to be having problems right now. Try again later.

**Comment by mrotteveel** [comment]
The use of email screwed up the link: the SourceForge status is at https://fosstodon.org/@sourceforgestatus

**Comment by aafemt** [comment]
> but given there have been no release of the ODBC driver in a long time, there are none available at GitHub.

My build is available: https://github.com/aafemt/firebird-odbc-driver/releases

**Reply by asperand87** [reply]
thank you, some detail please?

**Reply by aafemt** [reply]
What "detail"? You download either installer or archives. Installer is supposed to be run and do everything itself. The archives are supposed to be unpacked and then installation is done with `regsvr32 OdbcFb.dll`.
